__integrations__/utils/cosmosdb_model.test.ts

```diff
@@ -181,3 +242,33 @@ describe("find", () => {
 //     }
 //   });
 // });
+
+// describe("updateTTLForAllVersions", () => {
```

Comment:
this commented code can be removed?

Reply:
Removed in [4f31c4d](https://github.com/pagopa/io-functions-commons/pull/296/commits/4f31c4da58599e82e5ba5769adbbe0dd0a1affca)